tests/tr1/include/tfuns.h

```diff
@@ -3,6 +3,9 @@
 
 // common header for functional?.cpp
 
+#pragma warning(push)
+#pragma warning(disable : 5215) // '%s' a function parameter with volatile qualified type is deprecated in C++20

```

Comment:
The warning push/disable/pop is the only function change to this header; I made a non-functional drive-by fix to some weird formatting on lines 135-220.